actix-files/CHANGES.md

```diff
@@ -7,6 +7,8 @@
 * Bump up `percent-encoding` crate version to 2.1
 
 * Allow user defined request guards for `Files` #1113
+  
+* Add option to redirect to a slash-ended path `Files` #1132
```

Comment:
create new version entry, 0.1.5 is released already

Reply:
Which version would that be? 0.2.0 or 0.1.6?

Reply:
0.1.6

Reply:
Done! Didn't know the release date so I put a random date in the future.

Reply:
It may be confused, you can put `UNRELEASED` or something else.

Reply:
I changed it to TBD (to be defined)